README.md

```diff
@@ -80,6 +94,41 @@ end
 end # module UsesPreferences
 ```
 
+With the macros, all preferences are project-specific.
```

Comment:
Do you think it would be better to figure out how to pass kwargs to a macro, so we can have `@set_preferences!("foo" => "bar"; export=true)`, or something like that?

Reply:
I think that would make sense. A related issue: currently you have to read the docstring for `set_preferences!` in order to understand the `force` comment in the docstring for `@set_preferences!`. What do you want to do about that?

Reply:
Also, I don't understand `active_project_only` well enough myself to even try describing it in the README (I could read the code, but...)